Core/PoTItem.cs

```diff
@@ -783,6 +784,26 @@ public override void SaveData(TagCompound tag)
 		tag["affixes"] = affixTags;
 	}
 
+	public override void NetSend(BinaryWriter writer)
+	{
+		writer.Write((int)ItemType);
+		writer.Write((int)Rarity);
```

Comment:
Ideally, you'd cast Rarity here - and all other tiny enums or values - to byte instead of int when possible. It saves a small load on the network and is good habit.

Reply:
What about InternalItemLevel?
Its currently set to max 200 throughout the whole game, iirc.

Reply:
Then yeah, cast to byte. At worst, we change it to short if we intend on making it go to completely unreasonable heights.